usecases/classification/writer.go

```diff
@@ -127,7 +127,7 @@ func (r *batchWriter) saveObjects(items objects.BatchObjects) {
 	defer cancel()
 
 	if len(items) > 0 {
-		saved, err := r.vectorRepo.BatchPutObjects(ctx, items, nil)
+		saved, err := r.vectorRepo.BatchPutObjects(ctx, items, nil, 0)
```

Comment:
Do we need to include the version here?

Reply:
Discussed orally: Will check with core team to understand that usage.